### ISSUE TITLE
Controller is reloaded even when the view is in cache

### ISSUE THREAD

**Issue by svanson** [driftyco/ionic#2798]
Hi guys, 

I noticed when using navigation tips with ion-nav-view and ion-view a strange behavior.
I took your demo codepen "Tabs And Navigation" and added some logs and a controller of "Facts tab  " and another of "Facts2 tab" to explain myself.
Here is the link:
http://codepen.io/khor6475/full/vEXELY

When you go to "facts" tab and then go back to home, home controller is not reloaded and cached as described in your description .
Yet, if I then go back to "facts" tab, the controller is reloaded and cached view doesn't seem to work.
If I go to "Facts" tab  and then to "Facts2" tab and then go back to home, all the 3 views are cached and we can't see the logs saying the controller is loaded in the console.
In conclusion, it seems that each time user goes to previous view, cache of the old current view can't be accessed.
It seems stange to me, Is that the wanted behavior ?
I'm pretty new with angularJS and Ionic, so did I missed something which explains this behavior ?

Thanks !

**Comment by zeineb-Sidhom** [comment]
+1 
$ionicConfigProvider.views.forwardCache(true) 
fix my problem  :)

**Comment by FrankMN** [comment]
Thanks for sharing! it also solves my problem
